internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -196,6 +197,9 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
       boolean ssl = Boolean.parseBoolean(featureProperties.getProperty(SERVER_ENABLE_SSL, "false"));
       boolean isAutoJoin = Boolean.parseBoolean(featureProperties.getProperty(SERVER_IS_AUTO_JOIN, "false"));
       boolean isGrpcEnabled = Boolean.parseBoolean(featureProperties.getProperty(ENABLE_GRPC_READ_SERVER, "false"));
+      boolean isPlainTableEnabled =
+          Boolean.parseBoolean(configProperties.getProperty(ROCKSDB_PLAIN_TABLE_FORMAT_ENABLED, "false"))
```

Comment:
Just curious, why do we need both conditions to enable PT?

Reply:
Hmm, it is kind of weird indeed.
1. featureProperties contains the setup passed in.
2. configProperties contains user override in certain test. 

For this feature, (1) is 50% true for different server. 
(2) can be true when user add the override.
So we need to check both so to determine. 
Hmm, but I think I am wrong, it should be OR not AND I think.

Reply:
I think the part of the ordering is not correct indeed. We should always put default server config, and then put any config override from test. Let me fix them.